includes/polling/cisco-voice/cisco-iosxcode.inc.php

```diff
@@ -16,7 +16,7 @@
     $total = snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.7.1.0", null);
     $total = $total['1.3.6.1.4.1.9.9.86.1.7.1.0'][''];
 
-    if (isset($total) && ($total != "") && ($total != 0)) {
+    if (isset($total) && $total > 0) {
         // Available
         $available = snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.7.2.0", null);
         $available = $available['1.3.6.1.4.1.9.9.86.1.7.2.0'][''];
```

Comment:
You didnt change it, but down below I spot a bug:

dat_update($device, 'cisco-iosxcode', $tags, $fields);
should be:
**data_update**($device, 'cisco-iosxcode', $tags, $fields);

No one will be hitting this bug as on the calling code, the include is wrong:
includes/polling/cisco-voice.inc.php

include "cisco-voice/cisco-xcode.inc.php";
should be:
include "cisco-voice/cisco-**iosxcode**.inc.php";